CHANGELOG.md

```diff
@@ -1,204 +1,256 @@
 # Change Log
 
-## [v0.1.1](https://github.com/skippbox/kompose/tree/v0.1.1) (2016-10-06)
-[Full Changelog](https://github.com/skippbox/kompose/compare/v0.1.0...v0.1.1)
+## [v0.1.2](https://github.com/kubernetes-incubator/kompose/tree/v0.1.2) (2016-10-31)
+[Full Changelog](https://github.com/kubernetes-incubator/kompose/compare/v0.1.1...v0.1.2)
 
 **Implemented enhancements:**
```

Comment:
Oh, ok :-) We have to start  labeling all issues properly then.
It is little bin confusing as a lot of  "enhancements" are just in Closed issues section.